packages/api/endpoints/granules.js

```diff
@@ -133,7 +134,7 @@ async function del(req, res) {
   log.info(`granules.del ${granuleId}`);
 
   const granuleModelClient = new models.Granule();
-  const granule = await granuleModelClient.get({ granuleId });
+  const granule = await granuleModelClient.getRecord({ granuleId });
```

Comment:
Do we still need to support the `translateGranule` logic? If so, is it because we never provided a migration from old granule schema to new granule schema? I don't love having two separate methods for getting a granule

Reply:
> is it because we never provided a migration from old granule schema to new granule schema?

Correct.

> I don't love having two separate methods for getting a granule

I agree. Ideally, I think that `get` should return the raw record (like `getRecord` is doing now), and whoever called `get` could then have the option of calling `translateGranule` on the result if they wanted to. I'll make a note and see about coming back to that in an IP sprint.

Reply:
If the only benefit here is a speedup in one of our tests, I'd prefer to not add more code as a temporary solution (since it has a way of hanging around) and instead just wait until we can possibly make the entire fix at once. And in fact, since my work now focuses on granule deletion, I may be able to do this as part of my ticket